src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/models/CreatePayment.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.bulkscan.payment.processor.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotEmpty;
+import lombok.Data;
+
+import java.util.List;
+
+/**
+ * Represents the details to create a payment in payHub.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
If we have a value not expected, should we want a 400? I would assume if I added a json property called Mosh that we would get an error indicating it wasn't expected